Resources/Prototypes/Entities/Structures/Wallmounts/Signs/bar_sign.yml

```
@@ -31,14 +31,19 @@
       enum.WiresUiKey.Key:
         type: WiresBoundUserInterface
   - type: Appearance
+  - type: Damageable
+    damageContainer: Inorganic # This can be changed later.
```

```suggestion
  - type: Damageable
    damageContainer: StructuralInorganic
```
should take structural damage